apps/admin-gui-e2e/src/integration/resource-management-perun_admin.spec.js

```diff
@@ -0,0 +1,150 @@
+context('Actions', () => {
+  const dbVoName = 'test-e2e-vo-from-db-3';
+  const dbResourceName = 'test-e2e-resource-from-db-2';
+
+  const dbAddManager = 'test4';
+  const dbRemoveManager = 'resource-admin-2';
+
+  const dbAttributeToAdd = 'user-settings-name';
+  const dbAttributeToDelete = 'user-settings-description';
+
+  const dbGroupToAssign = 'test-e2e-group-from-db-2';
+  const dbGroupToRemove = 'test-e2e-group-from-db-1';
+
+  before(() => {
+    if (Cypress.env('BA_USERNAME_RESOURCE_MANAGER')) {
+      sessionStorage.setItem('baPrincipal', '{"name": "resourceManager"}');
+      sessionStorage.setItem('basicUsername', Cypress.env('BA_USERNAME_RESOURCE_MANAGER'));
+      sessionStorage.setItem('basicPassword', Cypress.env('BA_PASSWORD_RESOURCE_MANAGER'));
+      cy.visit('service-access');
+    }
+  });
+
+  beforeEach(() => {
+    cy.visit('organizations')
+      .get(`[data-cy=${dbVoName}]`)
+      .click()
+      .get('[data-cy=resources]')
+      .click()
+      .get('[data-cy=resource-list]')
+      .click()
+      .get(`[data-cy=${dbResourceName}]`)
+      .click()
+  })
+
+  it('test add attribute', () => {
+
+    cy.intercept('**/attributesManager/setAttributes/**').as('setAttributes')
+      .get('[data-cy=attributes]')
+      .click()
+      .get('[data-cy=add-attributes]')
+      .click()
+      .get(`[data-cy=${dbAttributeToAdd}-value]`)
+      .type('test')
+      .intercept('**/attributesManager/getAttributes/vo**').as('getAttributes')
+      .get('[data-cy=save-selected-attributes]')
+      .click()
+
+      // check that attribute was added
+      .get(`[data-cy=${dbAttributeToAdd}-value]`)
+      .should('exist')
+  });
+
+  it('test delete attribute', () => {
+
+    cy.intercept('**/attributesManager/removeAttributes/**').as('removeAttributes')
+      .get('[data-cy=attributes]')
+      .click()
+      .get(`[data-cy=${dbAttributeToDelete}-checkbox]`)
+      .click()
+      .get('[data-cy=remove-attributes]')
+      .click()
+      .intercept('**/attributesManager/getAttributes/vo**').as('getAttributes')
+      .get('[data-cy=delete-attributes]')
+      .click()
+
+      // check that attribute was deleted
+      .get(`[data-cy=${dbAttributeToDelete}-checkbox]`)
+      .should('not.exist')
+  });
+
+  it('test add resource manager', () => {
+
+    cy.intercept('**/authzResolver/setRole/**').as('setRole')
+      .get('[data-cy=advanced-settings]')
+      .click()
+      .get('[data-cy=managers]')
+      .click()
+      .get('[data-cy=add-manager-button]')
+      .click()
+      .get('[data-cy=search-manager-input]')
+      .type(`${dbAddManager}`)
+      .get('[data-cy=search-manager-button]')
+      .click()
+      .get(`[data-cy=${dbAddManager}-checkbox]`)
+      .click()
+      .get('[data-cy=add-manager-button-dialog]')
+      .click()
+      .intercept('**/authzResolver/getRichAdmins**').as('getRichAdmins')
```

Comment:
You should intercept the response before clicking on the button (and also wait for it).
```suggestion
      .intercept('**/authzResolver/getRichAdmins**').as('getRichAdmins')
      .get('[data-cy=add-manager-button-dialog]')
      .click()
      .wait('@getRichAdmins')
```